lib/matplotlib/tests/test_lines.py

```diff
@@ -343,3 +352,25 @@ def test_input_copy(fig_test, fig_ref):
     # Trigger cache invalidation
     l.set_drawstyle("steps")
     fig_ref.add_subplot().plot([0, 2, 4], [0, 2, 4], ".-", drawstyle="steps")
+
+
+@check_figures_equal(extensions=["png"])
+def test_markevery_prop_cycle(fig_test, fig_ref, cases):
```

Comment:
```suggestion
def test_markevery_prop_cycle(fig_test, fig_ref, cases):
    """Test that we can set markevery prop_cycle."""
```
To make it more clear to the reader what the test does.

Reply:
Thanks, added!